fixtures/QuarkChainStateTests/CallPrecompile.json

```diff
@@ -0,0 +1,109 @@
+        },
+        "post" : {
+            "ConstantinopleFix" : [
+                {
+                    "_comment": "This creates a tx with transfer token id 0x1234 that queries its token id",
+                    "hash" : "0x35d2ba3fb97dfebd28c376860e9bb0bb6231deed43318cf2cc53667c0eb484db",
+                    "indexes" : {
+                        "data" : 0,
+                        "gas" : 0,
+                        "value" : 0,
+                        "transferTokenId": 1
+                    },
+                    "logs" : "0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347"
+                },
+                {
+                    "_comment": "This creates a tx with transfer token id 0x1234 that does not query its token id",
+                    "hash" : "0x82203068f3d4ea4b0549f6aa5421228aafb8685bace998efc97bbfb908e21256",
+                    "indexes" : {
+                        "data" : 1,
+                        "gas" : 0,
+                        "value" : 0,
+                        "transferTokenId": 1
+                    },
+                    "logs" : "0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347"
+                },
+                {
+                    "_comment": "This creates a tx with default token id that queries its token id",
+                    "hash" : "0xa704fc7f1095fcb8aeaf80b2bcea3683c88c130884e7508549fdce8717f5a485",
+                    "indexes" : {
+                        "data" : 0,
+                        "gas" : 0,
+                        "value" : 0,
+                        "transferTokenId": 0
+                    },
+                    "logs" : "0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347"
+                },
+                {
+                    "_comment": "This creates a tx with default token id that does not queries its token id",
+                    "hash" : "0xa9a6ae2ec20a8a1207ca332ef074bc78fba7cc7eadb11c17130a4e1f4426528a",
+                    "indexes" : {
+                        "data" : 1,
+                        "gas" : 0,
+                        "value" : 0,
+                        "transferTokenId": 0
+                    },
+                    "logs" : "0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347"
+                }
+            ]
+        },
+        "pre" : {
+            "0xa94f5374fce5edbc8e2a8697c15331677e6ebf0b" : {
+                "_comment": "This is the sender contract's address",
+                "balances" : {
+                    "0x8bb0": "0x0de0b6b3a7640000",
+                    "0x1234": "0x0de0b6b3a7640000"
+                },
+                "code" : "",
+                "nonce" : "0x00",
+                "storage" : {
+                }
+            },
+            "0x0c2e77121daf0270d26bf0a7e9ab0faa8bf739ef" : {
+                "_comment": "This is a contract with a function to call the precompiled contract PROC_CURRENT_MNT_ID",
```

Comment:
I prefer to have the contract written somewhere, an example from existing test case in ethereum/tests looks like following

```
"comment" : "The first test case required here https://github.com/ethereum/tests/issues/431#issue-306081539 .\nInvoked Solidity 0.4.21 with solc --bin-runtime --optimize\ncontract DoubleSelfdestructTest {\n  address constant B1 = 0x1;\n  address constant B2 = 0x2;\n  function remoteSelfdestruct() external {\n    selfdestruct(B2);\n  }\n  function() external payable {\n    this.remoteSelfdestruct();\n    selfdestruct(B1);\n  }\n}\n",
```